compiler/x/i386/codegen/TreeEvaluatorTable.hpp

```diff
@@ -564,9 +564,9 @@
    TR::TreeEvaluator::badILOpEvaluator,                    // TR::monent (J9)
    TR::TreeEvaluator::badILOpEvaluator,                    // TR::monexit (J9)
    TR::TreeEvaluator::badILOpEvaluator,                    // TR::monexitfence (J9)
-   TR::TreeEvaluator::badILOpEvaluator,                  // TR::tstart (J9, WCode)
-   TR::TreeEvaluator::badILOpEvaluator,                 // TR::tfinish (J9, WCode)
-   TR::TreeEvaluator::badILOpEvaluator,                 // TR::tabort(J9, WCode)
+   TR::TreeEvaluator::tstartEvaluator,                  // TR::tstart (J9, WCode)
```

Comment:
Why the mentions of WCode here? Why the mentions of J9? This is generic OMR code right?